zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/IncidentControllerTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.controller;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.when;
+
+import io.camunda.service.IncidentServices;
+import io.camunda.service.security.auth.Authentication;
+import io.camunda.zeebe.gateway.rest.RestControllerTest;
+import io.camunda.zeebe.protocol.impl.record.value.incident.IncidentRecord;
+import java.util.concurrent.CompletableFuture;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+
+@WebMvcTest(IncidentController.class)
+public class IncidentControllerTest extends RestControllerTest {
```

Comment:
❓ Since we only have the key here, should we add tests that break the API to ensure we're covered? Providing no key is not really possible (you'll get a 404 but, meh 🤷). We could test with a String like `abc` instead of a number? One thing we should do is test that the incident was not found and we receive an appropriate answer.

Reply:
Yeah, I agree on "let's not test the framework" but disagree on "I trust the framework" (personally) 😄

Reply:
Yeah, I shouldn't 🤣

Reply:
As long as it's Spring, you shouldn't 😆